ansible_base/activitystream/serializers.py

```diff
@@ -20,6 +22,7 @@ class Meta:
 
     content_type_model = serializers.SerializerMethodField()
     related_content_type_model = serializers.SerializerMethodField()
+    changes = serializers.SerializerMethodField()
```

Comment:
I don't often see a `SerializerMethodField` that's the same name as the field, but since it works that seems like the best for this problem.